js/main.js

```diff
@@ -108,6 +109,7 @@ function swapHeaders(event) {
 
 function openPopUp(event) {
   $popUpContainer.className = 'popUpContainer row justify-center align-center';
+  $delete.style.visibility = 'hidden';
```

Comment:
You should have a class you can add or remove rather than setting inline styling like this.